server/service/src/auth.rs

```diff
@@ -524,6 +527,14 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
         Resource::QueryAsset,
         PermissionDSL::HasPermission(PermissionType::AssetQuery),
     );
+    map.insert(
+        Resource::QueryDemographic,
+        PermissionDSL::HasPermission(PermissionType::ServerAdmin),
+    );
+    map.insert(
+        Resource::MutateDemographic,
+        PermissionDSL::HasPermission(PermissionType::ServerAdmin),
```

Comment:
Note have these permissions linked to serverAdmin currently.

Are permissions required? 

I wonder if a simple check for isCentralServer would be enough, and then could remove permission checks?
Given that this is not involve core business logic.

Reply:
I think there needs to be some permission, but not sure exactly what the plan is at this stage...
This if fine for now...